editoast/src/views/v2/train_schedule.rs

```diff
@@ -183,14 +185,46 @@ async fn get(
     let train_schedule_id = train_schedule_id.id;
     let conn = &mut db_pool.get().await?;
 
-    // Return the timetable
     let train_schedule = TrainSchedule::retrieve_or_fail(conn, train_schedule_id, || {
         TrainScheduleError::NotFound { train_schedule_id }
     })
     .await?;
     Ok(Json(train_schedule.into()))
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, IntoParams)]
+struct GetBatchParams {
+    ids: Vec<i64>,
+}
+
+/// Return a specific train schedule
+#[utoipa::path(
+    tag = "train_schedulev2",
+    params(
+        ("ids" = Vec<i64>, Query, description = "Ids of train schedule"),
+    ),
+    responses(
+        (status = 200, description = "Retrieve a list of train schedule", body = Vec<TrainScheduleResult>)
+    )
+)]
+#[get("")]
+async fn get_batch(
+    db_pool: Data<DbConnectionPool>,
+    params: QsQuery<GetBatchParams>,
+) -> Result<Json<Vec<TrainScheduleResult>>> {
```

Comment:
Looking at this, I don't understand the name `TrainScheduleResult`. From what I understand, it's the object returned in a view. The current file is in module `views/v2` so I would just name it `TrainSchedule` since it's full path already makes it clear that it's a view (`views::v2::TrainSchedule`).

And the other imported `modelv2::train_schedule::TrainSchedule` can be either always used as `modelv2::train_schedule::TrainSchedule` (by the way, maybe we should make it so we can use it as `modelsv2::TrainSchedule` directly), or imported as `use crate::modelsv2::train_schedule::TrainSchedule as TrainScheduleModel`?

Reply:
Not directly related to the current PR, so I'm resolving. We'll discussed later if something need to be done later.